packages/components/src/slot-fill/stories/index.js

```diff
@@ -33,23 +30,21 @@ export const _default = () => {
 			</p>
 			<Fill name="name">Grace</Fill>
 			<Fill name="age">33</Fill>
-		</Provider>
+		</SlotFillProvider>
 	);
 };
 
 export const withFillProps = () => {
-	const name = text( 'name', 'Grace' );
-	const age = number( 'age', 33 );
```

Comment:
Controls aren't very helpful for these components so I think we can remove them altogether. Separate stories with good usage examples and descriptions are a better fit. (We can add better descriptions in the full TypeScript refactor.)